components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -326,4 +328,63 @@ - (void)testAdjustFontForContentSizeCategoryWhenScaledFontIsUnavailableSetsTheAl
   XCTAssertFalse(self.alert.alertView.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable);
 }
 
+/**
+ Test legacy dynamic type has no impact on a @c MDCButton when @c
+ adjustFontForContentSizeCategoryWhenScaledFontIsUnavailable is set to @c NO that the font stays
+ the same.
+ */
+- (void)testLegacyDynamicTypeDisabled {
+  // Given
+  UIFont *fakeTitleFont = [UIFont systemFontOfSize:55];
+  self.alert.titleFont = fakeTitleFont;
+  self.alert.mdc_adjustsFontForContentSizeCategory = YES;
+
+  // When
+  self.alert.adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable = NO;
+  MDCAlertControllerView *view = (MDCAlertControllerView *)self.alert.view;
```

Comment:
Move this line down to the "then" section.

Reply:
Done